arangod/Sharding/ShardDistributionReporter.cpp

```diff
@@ -201,14 +201,14 @@ static void ReportPartialNoProgress(
     VPackBuilder& result) {
   // create a sorted list of shards, so that the callers will get the
   // shard list in a deterministic order (this is very useful for the UI).
-  containers::SmallVector<std::string_view, 8> sortedShards;
+  containers::SmallVector<ShardID, 8> sortedShards;
   sortedShards.reserve(shardIds.size());
 
   for (auto const& s : shardIds) {
     sortedShards.emplace_back(s.first);
   }
   // sorts sortedShards in place in a deterministic order
-  ShardingInfo::sortShardNamesNumerically(sortedShards);
+  std::sort(sortedShards.begin(), sortedShards.end());
```

Comment:
Why not make `sortedShards` a sorted container in the first place?

Reply:
Done